stable/cockroachdb/templates/cockroachdb-petset.yaml

```diff
@@ -80,13 +96,28 @@ spec:
         component: "{{.Release.Name}}-{{.Values.Component}}"
       annotations:
         helm.sh/created: {{.Release.Time.Seconds | quote }}
-        pod.alpha.kubernetes.io/initialized: "true"
+        scheduler.alpha.kubernetes.io/affinity: >
```

Comment:
I guess more explicitly, what happens to this in clusters that dont have alpha features?